core/signer/signer.go

```diff
@@ -257,21 +289,32 @@ func (s *Signer) CreateEpoch(ctx context.Context, forceNewEpoch bool) error {
 		len(mutations), len(leaves))
 
 	// Set new leaf values.
+	mapSetStart := time.Now()
 	setResp, err := s.tmap.SetLeaves(ctx, &trillian.SetMapLeavesRequest{
 		MapId:  s.mapID,
 		Leaves: newLeaves,
 		MapperData: &trillian.MapperMetadata{
 			HighestFullyCompletedSeq: seq,
 		},
 	})
+	mapSetEnd := time.Now()
 	if err != nil {
 		return err
 	}
 	glog.V(2).Infof("CreateEpoch: SetLeaves.HighestFullyCompletedSeq: %v", seq)
 
 	// Put SignedMapHead in an append only log.
-	if err := s.sths.Write(ctx, s.logID, setResp.MapRoot.MapRevision, setResp.MapRoot); err != nil {
+	// TODO: sths.Write currently hangs.
```

Comment:
`// TODO: sths.Write currently hangs.`
Is this still true? If yes, change to TODO(name):